mkosi/machine.py

```diff
@@ -117,6 +117,10 @@ def build(self) -> None:
         if needs_build(self.config):
             check_root()
             check_native(self.config)
+
+            # Useful if testing within Docker
+            if parse_boolean(os.getenv("MKOSI_TEST_NO_NAMESPACE", "0")):
+                raise unittest.SkipTest("Build test skipped due to environment variable.")
```

Comment:
Let's drop this and do it in a separate PR if you really want it included

Reply:
See #1179